inc/gui.class.php

```diff
@@ -528,13 +528,13 @@ public static function options_page() { ?>
 							<a href="https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=TD4AMD2D8EMZW" target="_blank" rel="noopener noreferrer"><?php esc_html_e( 'Donate', 'antispam-bee' ); ?></a>
 						</p>
 						<p>
-							<a href="<?php echo esc_url( __( 'https://wordpress.org/plugins/antispam-bee/faq/', 'antispam-bee' ) ); ?>" target="_blank" rel="noopener noreferrer"><?php esc_html_e( 'FAQ', 'antispam-bee' ); ?></a>
+							<a href="<?php echo esc_url( __( 'https://wordpress.org/plugins/antispam-bee/#faq', 'antispam-bee' ) ); ?>" target="_blank" rel="noopener noreferrer"><?php esc_html_e( 'FAQ', 'antispam-bee' ); ?></a>
```

Comment:
Wouldn't it be an option to hardcode this link as well, @2ndkauboy? Similar to what you did with the other link below? I think it's very unlikely that the FAQ link will change in the near future.

Reply:
This link is actually translated:
![image](https://user-images.githubusercontent.com/6458412/195341161-1a75af3a-b59a-45e1-a896-4a506a1037b4.png)

But there is no supported wp.org support forum for the plugin in another language. I assume this is why the translate function has been removed.

Reply:
Ah, I see. Thanks for clarifying this, @websupporter.